BREAKING.md

```diff
@@ -80,6 +82,21 @@ The following types in the @fluidframework/container-loader package are not used
 -   IPendingContainerState
 -   ISerializableBlobContents
 
+### Closing Container no longer disposes
```

Comment:
Do you need to do this in addition to the changeset?  I haven't walked through this new process yet

Reply:
@markfields I think the plan is to replace BREAKING.md with changesets. @tylerbutler Please correct me if that's wrong